composer.json

```diff
@@ -33,7 +33,6 @@
         "contao/imagine-svg": "^1.0",
         "dantleech/phpcr-migrations-bundle": "^1.3",
         "doctrine/annotations": "^1.2",
-        "doctrine/cache": "^1.0",
```

Comment:
The `doctrine/phpcr-bundle` still installs `doctrine/cache` `^1.0`.

Reply:
no longer by phpcr-bundle, but by a lot of other parts of doctrine (at least those would allow ^2.0 too): 
```
$ composer why doctrine/cache
sulu/sulu                3.0.x-dev requires doctrine/cache (^1.0.1)            
doctrine/dbal            3.6.6     requires doctrine/cache (^1.11|^2.0)        
doctrine/doctrine-bundle 2.10.2    requires doctrine/cache (^1.11 || ^2.0)     
doctrine/orm             2.15.5    requires doctrine/cache (^1.12.1 || ^2.1.1) 
doctrine/persistence     2.5.7     requires doctrine/cache (^1.11 || ^2.0) 
```